packages/css/src/components/accordion/accordion.template.ts

```diff
@@ -0,0 +1,60 @@
+import { Accordion } from '@dso-toolkit/sources';
+
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+
+import { accordionHandleHeading } from './accordion-handle-heading.template';
+
+export function accordionTemplate({ variant, sections, reverseAlign }: Accordion) {
+  return html`
+    <div class="dso-accordion ${classMap({
+      [`${variant}`]: !!variant,
+      'dso-align-viewer': !!reverseAlign
+    })}">
+      ${sections.map(section => html`
+        <div class="dso-accordion-section ${classMap({
+          [`dso-${section.state}`]: !!section.state,
+          'dso-open': !!section.open,
+          'dso-nested-accordion': !!section.sections
```

Comment:
Een lege array is nog steeds truthy. Bij een lege array wordt de class `dso-nested-accordion` gezet.